### PR TITLE
Fixed issues with NETIME, H3TEXT, and TIMES found in #2225

### PR DESCRIPTION
**Pull request by eswenson1**
Added LOCAL-TIME-SERVER as CNAME for BRIDGE in H3TEXT.
Fixed NETIME to update comment associated with MINQ.  Removed ROUTER, CHAOS2, TT, and the not-globally-known ITSes.
Fixed TIMES to remote ROUTER, CHAOS2, TT, and the not-globally known ITSes.
Fixed TIMES to set TCPP back to -1 (true) since using a JNAME of CTIMES will use chaosnet.

### CODE REVIEW

**Review by larsbrinkhoff — APPROVED** [review]

**Comment by eswenson1** [comment]
Thanks, @larsbrinkhoff. This time, however, I won’t make the same mistake as last time — I’ll also wait for @bictorv ’s approval! :-)

File: build/h3text.2018
Comment (bictorv) [inline build/h3text.2018:27]: I think it would be useful with a comment in the file, saying essentially "please make sure to update this address if/when you join the global Chaosnet"?

Comment (eswenson1) [reply build/h3text.2018:27]: Good idea. I’ll update.

**Comment by larsbrinkhoff** [comment]
The "simh" build target has been giving us trouble since a few weeks back or so.  Which is strange, since before then it was quite stable and I haven't updated the subrepository.  Anyway, the build often has to be restarted a few times and then it's ok.

**Comment by larsbrinkhoff** [comment]
I folded the last commit into the first, and fixed your email address, Eric.

**Comment by eswenson1** [comment]
Thanks. What was wrong with the email address?

**Comment by eswenson1** [comment]
Any way to figure out, now, what it was? I suppose it would be gone from the git log?

**Comment by larsbrinkhoff** [comment]
I have it in the "reflog".  I'll email you.

**Comment by larsbrinkhoff** [comment]
All approved!

**Pull request merged**
